src/features/expense/ui/UnReviewedExpenseListItem/UnReviewedExpenseListItem.test.tsx

```
@@ -0,0 +1,35 @@
+import { render } from '@testing-library/react';
+import { format } from 'date-fns';
```

ESLint

[error] 'format' is defined but never used.

(@typescript-eslint/no-unused-vars)